fn/client/src/components/login/Login.js

```diff
@@ -50,7 +50,7 @@ const Login = ({ storeService, setUserLogged, setUserLoading, userLogged, userLo
     const setInitialCart = (res) => {
         const { accessToken, refreshToken, userId } = res
         const cart = { cartNumbers: 0, cartProducts: [] }
-        axios.put(`https://lv487node-backend.herokuapp.com/users/cart/${userId}`, { cart });
+        axios.put(`http://localhost:5000/users/cart/${userId}`, { cart, "x-auth-token": accessToken });
```

Comment:
don use hard code basicUrl
it should be in config!
here use only path without basicUrl

Reply:
fixed